### PR TITLE
Frontier Psychiatrist - Two new psionic abilities to help with sanity

### PR DESCRIPTION
**Pull request by AysheDaArtist**
## About The Pull Request
<details>
<summary>
	Adds two new psionic abilities

For all psionics:
Meditative Focus - Expend two psi points of your psi essence to focus your mind and increase your sanity, scales off your Cog stat and Psi focused Background perks ( Scholar / Bedlam )

For psychiatrist/enhanced psion:
Psionic Tranquility - Expend two psi points of your psi essence to calm the mind of another person you are grappling and restore their sanity, scales off your Cog stat and Psi focused Background perks ( Scholar / Bedlam )

Dev note: This has been tested and balanced with LongSleeves/Landen to not cause issues with inspiration system and is capped to (max sanity level - 10) and costing 2 psi points to curb Power gaming abuse.  If issues arrise, a timer will be coded in to prevent inspiration abuse
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by DorfGuy1** [comment]
this is a cool idea. Can the soteria psi have a buffed version, that way the person who's in fluff supposed to handle mental things can handle the mechanical mental things as well?

**Comment by AysheDaArtist** [comment]
Thank you Lilly!
psi_points = psi_point_cost
changeLevel
Wrote these two down for future use, thank you!

**Comment by AysheDaArtist** [comment]
Lilly, I tested your changes:

psi_points = psi_point_cost will change your Psi essence to 2 if you're over 2

If you try to use it past 90 sanity with more than 2 Psi essence, you are dropped back down to 2.

**Comment by AysheDaArtist** [comment]
psi_points = psi_points + psi_point_cost

That should fix it :)

File: code/modules/psionics/psion_powers/healing_powers.dm [hunk 1]
```diff
@@ -71,3 +71,30 @@
...
+// Heals sanity
+/obj/item/organ/internal/psionic_tumor/proc/meditative_focus()
+	set category = "Psionic powers"
+	set name = "Meditative Focus (2)"
+	set desc = "Expend two psi points of your psi essence to focus your mind and increase your sanity."
+	psi_point_cost = 2
+
+	if(pay_power_cost(psi_point_cost))
+		if(owner.sanity.level >= (owner.sanity.max_level - 10))
+			psi_points = psi_point_cost
```
Comment (Trilbyspaceclone) [inline code/modules/psionics/psion_powers/healing_powers.dm:84]:
```suggestion
			psi_points += psi_point_cost
```

**Pull request merged**